test/elixir_analyzer/exercise_test/common_checks_test.exs

```diff
@@ -266,6 +266,32 @@ defmodule ElixirAnalyzer.ExerciseTest.CommonChecksTest do
         end
       ]
     end
+
+    test_exercise_analysis "reports Kernel.dbg",
+      comments: [Constants.solution_debug_functions()] do
+      [
+        defmodule MyModule do
+          def foo() do
+            (1 + 1)
+            |> dbg()
+          end
+        end,
+        defmodule MyModule do
+          alias Kernel, as: Debug
+
+          def foo(name) do
+            Debug.dbg(name)
+          end
+        end,
+        defmodule MyModule do
+          import Kernel, only: [dbg: 1]
```

Comment:
Not sure how legit this test is. There are two `dbg` versions: `Kernel.dbg/2` and `Kernel.dbg/3`. I wonder what this import does, I think it imports `Kernel.dbg/2` but I might be wrong

Reply:
There are three versions of `Kernel.dbg`, with 0, 1, and 2 arguments. You can see that in the docs for [`Kernel.dbg/2`](https://hexdocs.pm/elixir/Kernel.html#dbg/2) - technically the docs only list a 2 argument variant, but both arguments are defined as optional, so we know for sure 0 and 1 arity versions must exist too 🙂.

`dbg/3` is from another module, it's from `Macro`: https://hexdocs.pm/elixir/Macro.html#dbg/3 and it's not related to this ticket.

Anyway,  calling `import Kernel, only: [dbg: 1]` has a really fun side effect - it un-imports all of the other functions and macros imported from Kernel 😁 for example:

```elixir
iex()> import Kernel, only: [dbg: 1]
Kernel
iex()> 2 + 3
error: undefined function +/2 (there is no such import)
  iex:4

** (CompileError) cannot compile code (errors have been logged)
```

This error happens because the `+/2` macro is normally available because the whole `Kernel` module is usually imported by default.

What does all of this mean for this PR? We don't really need a test for importing `Kernel.dbg`. The first two test cases are enough 👍

Reply:
Oh my, thanks for the followup 🙇